test/Be.Vlaanderen.Basisregisters.Shaperon.Tests/ShapeBinaryWriterTests.cs

```diff
@@ -26,15 +26,18 @@ public ShapeBinaryWriterTests()
                 customization.FromFactory(generator =>
                     {
                         var result = ShapeType.NullShape;
-                        switch (generator.Next() % 3)
+                        switch (generator.Next() % 4)
                         {
                             //case 0:
                             case 1:
-                                result = ShapeType.Point;
+                                result = ShapeType.PointM;
```

Comment:
Why PointM? PointM only makes sense as part of a PolyLineM. PointM is on its way out so not really that big a deal.

Reply:
My bad. Added commit to fix.